### ISSUE TITLE
SPC f e R fails on develop branch

### ISSUE THREAD

**Issue by dubnde** [syl20bnr/spacemacs#6429]
#### Description :octocat:
`SPC f e R` fails on `develop` branch. Appears related to commit https://github.com/syl20bnr/spacemacs/commit/052142ab331c67a092b33b27a92917008e594d02

#### Reproduction guide :beetle:
- Start Emacs so it generates a new `.spacemacs` file
- Run `SPC f e R` key sequence to sync the configuration

*Observed behaviour:* :eyes: :broken_heart:
Fails to resync with following error

```
** TEST: Each element of dotspacemacs-startup-lists includes only 'recents, 'bookmarks or 'projects
*** FAIL: (recents . 5)
*** FAIL: (projects . 7)
```

*Expected behaviour:* :heart: :smile:
`SPC f e R` should run without errors

#### System Info :computer:
- OS: darwin
- Emacs: 24.5.1
- Spacemacs: 0.105.21
- Spacemacs branch: develop (rev. 878c45a)
- Graphic display: t
- Distribution: spacemacs-base
- Editing style: vim
- Completion: ivy
- Layers:
```elisp
(ivy emacs-lisp)
```

**Comment by bmag** [comment]
Thanks, indeed it seems we need to update some tests to match the changes in https://github.com/syl20bnr/spacemacs/commit/052142ab331c67a092b33b27a92917008e594d02.
cc @ralesi

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
